source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -97,6 +108,22 @@ content: |
            
            BsonBinary dataKeyId = clientEncryption.createDataKey("aws", dataKeyOptions);
            String base64DataKeyId = Base64.getEncoder().encodeToString(dataKeyId.getData());
+     .. tab::
+        :tabid: nodejs
+
+        .. code-block:: javascript
+        
+           const encryption = new ClientEncryption(client, {
+               keyVaultNamespace,
+               kmsProviders
+           });
+           encryption.createDataKey('aws', {
+               masterKey: { 
+                 key: <Master Key ARN>,// e.g."arn:aws:kms:us-east-2:111122223333:alias/test-key"
+                  region: 'us-east-1' 
+               },
+               keyAltNames: ['key1']
+           })
```

Comment:
nits:
- Let's put the placeholder in quotes if it's a string, i.e. `key: "<Master Key ARN>",`
- 10 spaces -> 11 spaces

Reply:
supernit: we should try to consistently use double-quotes rather than mixing single and double

This comes for free if you run code through a formatter (Prettier for JS). The only real reason we'd want to actively mix single and double quotes is if we have quotation marks in a string. In that case we'd use whatever made us literally escape characters the least.

Reply:
Fixed all requested changed ^